lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -11,6 +11,10 @@ class Base
           ::RubyParser::SyntaxError,
         ]
 
+        DEFAULT_MASS_THRESHOLD = 28
+        BASE_POINTS = 1_500_000
+        POINTS_PER = 50_000
```

Comment:
I don't think it's any more confusing than using the inherited methods, which I don't find confusing at all.